### ISSUE TITLE
06/20/24 Retail Update Affecting Existing Auras in strange way

### ISSUE THREAD

**Issue by OldWorldRaiders** [WeakAuras/WeakAuras2#5173]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues.

### Description

Ever since the Retail (Dragonflight) update, to 5.14.0, one of my weakauras is behaving in a strange way. I have an aura that shows Avenging Wrath's icon and remaining buff duration while I have the buff, that would then inverse/grey out the icon and show the remaining cooldown on it. Ever since the update, when the buff goes away and it _should_ show the inverse icon + CD remaining it just shows the icon (non inverse) with no CD remaining.

I uploaded this one here: https://wago.io/6nS4lsAMz

Other weakauras are acting in a similar way. When I revert to WeakAuras addon version 5.13.2-55-g5784e0e OR 5.13.2-56-g81e4086 it works like normal. 

### WeakAuras Version

5.14.0

### World of Warcraft Flavor

Retail (Default)

### World of Warcraft Region

US/NA

### Tested with only WeakAuras

I got this issue with only WeakAuras enabled

### Lua Error

```Text
N/A
```

### Reproduction Steps

Activate Avenging Wrath while in combat, and stay in combat through the duration of the buff. 

### Last Good Version

5.13.2-55-g5784e0e AND 5.13.2-56-g81e4086

### Export String

```Text
!WA:2!1DvxSX1r1FVEPYw(HQKnutsudySiMKuIjE3eRKqtd7AVU2vo2U3DtCsjWE)A29EJV7DMmZCT9gvGMTuQdqiWw67HvcjEGhqROpa)1FEWpWZNSQsrcrijwqri1NYBi1QWzM7DDCOnxlFVZCMZC(AoZVZztntM6zCZ4Ur22(o0Ws0iUd5f7ZEnRA05MUfFLPoXs1fN7Ago0ak)1sJpTDIesA9YK1L9ZBSy1QcI8Yd2Xk0XJYxI6hkTNQ4cLlA01HsdCPRfwAnFgXKTRNSXZZCTKe7i9NSr8G(5wosFAOysdH0IlVa3LwPwaDnVoQ3vIxTGNVlPjUIGgf661chfltV2AUQYTQty7zzI1k5J4wItVm3s6viQAv7Q(H(cVHHmEfWrYXSJ3y)WE8Ak5(1Qr4IV3y8KHBSNwwOacrXjoTD2CtCYtDcd89jpUDoCyUtXr3AXWGgMgAEoc7aWVg6CNBDNBaFa8)J)97VZTgb6a)z4pQMW2h8E43pa6mc8NW1El8)M40)V7CRorcsfjgPK(mV22OTwYJU2IHBk0FYJo(QeoY0cOImFKIBxIITPcSecVTer2KvjHYsij)17uzQ8LkxPu58gL3c59v50iwfhmCj9m08zplXkq61owiLBWi8zlo)sZC(5leHbgBwGvdcVPl6(QqEMjAkyKGG5CfhzVPVXDh8n)zfK4MmuXNSDRBjD8iIytgo4wo062(HKZftUTYhMNUgri5rHA1Bi9RtC32QxItqR2O0sfNF(wOrgh(hAmy)BSxdjMuSsbRijfMiZeDRrcjCFNyDPfD(G1SAiGrgshevuQ546XEHEZcOcbTkMxl50aVMkYkF0SL2Lubu0DGVjCmiNHMuN4SIkQ8m4uWHGd3UNSwm0SRAmNu3cZGcR5bVeBFtLKIpYsCAnorig5qLuc6WWjHxPlhd1L6PQTYJoCnCJJOZk1cBB7WeYg7eoEw8AeHhCA4BX2TIYt52MT0PlQaJjCuyC4LhZo26GVnmn6nDC9fxjkuN1K1kSbB3w6XLJJSNJ6s(D99i8skHhAfCbmRhpKF)hTcHWYJwJJ0qDSB2Yk0VUobys45VaKRqinK0KyjiQmgy5nvdlj5Ohj9gOBVSLkQudUGGHCxHnddiezHQwUKcQG2mWLgaYbldVb8DGlIFh(c6PQH4kkQxg(UyMPWsQeiXJVASbMUnMWyqQPqiUYWqUU4uBmguJRacQ2XMYDj8kc)Rrs3nzcvJmLEZKPXayPsLk9wjuw1x4Bhqm7TBIBncB3LUAKfNmYmrbbJSSNVKa5AP0MMNpQRcNzgo5QrKqNgPGCCCnfTnvVMojmKUPAMks11oskPHlI(bEVsJsnVoMDynU2LIHpt3snzQyi2zp7)4pOE(nZE2pSS6PsAnVxmH3nvxR7XUN2Gk757SsiM9LUGgZulUsowbKyjppELuCgT2lO9dVpcUYsPp24tarEqqAOUXs(Rtca6laSuWvvgXh)jF6x5Xp(XZE2)1DV7d6RV9NgeMGmnSAAGJ(nScS2UH1n3yywgjIOJ3rSC32itX2NMO(vvkhtLQisgal3A71g9Gc2U2X2N6jhsXujbv1LuAwEXLmM7vNTmBp6fuGcQ8tNL9Dre(840hfRm0pC1eVE34ssjX4uBQx)k4fC)QnGbBZPsmftvnQWclUqX4Jsfpjou8UtCObARP507i6J)e85tpGo6Gp)N0TsC1qz7IRRs8fi00tjPe7yGUAARHNdioaR5YjdInVE5KjrLDuuf(HpvGUNx1DB9wct(py3NYQ3XQAG(fpF55NBHIBSp2x(ZC8WQOqMtMLcEZrpiJnYZIR6u3kQlOMWpOq(ZxEr4TGRdnVo7R9S2qaPMLtJkvdOuUj8JGbH3b(XSV6ZIDma64RV4JzAVl8EPGB85g1HFc8tHRLc(LPGFf8(WnHFoClt4xGI)TtbFFDIe06aSd)S0JBdS8TVtfPhEI5Hq5P)CsCz9sCpLThXVMN81leqTC34lXrWCv91HnQhHLVhsJnRkZX9yz0G5Qs04HORcO3XR120mlGi4KHHBpuZKYaoEfuOxksB90Bdb4r9eqUjC7jVP5BB(oMCjEZouMvXSwMbOod8SJjJuh7Lm)6OkObA5vqlf43oaC7N3CatdT4X58LYpF(PNBbKfDXigrABjLbeS5aFSzch0PAOnOTlvTsiwRlUo4Qee0jGCEFpE14UZ0MdBv22MXq6INQwcSKMTFIaWYu8RH1tWEk6NTlKN8UQRLexI75SwhBPbVxVsJEGPE2cDxP2(1zuUSJFy8XbU0yDWHoKIQojeMS96xlKYjlY0DsQPwKZPCHPQQOQXM5WctSdnNQ6hA0KlRMlUSUeDLzPbnQKuJMOlrZxp(A0xyWTyjL3JBq(iVyFFgd3p0RnxxHsDvRGQB6Sr(UD8RwS2fQhn9HU0H2SxdXfXknEgRPqPEDdRaMNv6owjIcfu)D0nYILxTKwdu4AuADfyDo4vmzFXEYqbCnTVWcrlCnTfK6yANXeJFSXpwBj1jPU5Et9U0r9D5tHnqIvHFIsSwVFDkAV(xW(cmu3NFum(Xmk9R8d7stzuS4cDr8wKCsKDiB05Q6xBi8cGsrimWFXSTQMVVE93ymdS7pNvMe2)Z1CvlUVYiBtd7PmJvTcIiP5yVoHyVo7Fm4(MnXimIqjBavgd(B4o)g9b3B0fnS1ssC6zuugLYY8YW9YMFHPHTocs65IPC)mta3BlY6m)4uMY4DBLYH7b)D4(Pr5NgUxbvFsXdH)j8aTwHhQASy8CJRknI6ffzAuKJEMZeltB8hJGAh5Ey4HSmkEpX47eOoEpBBgzrzpXo5EY)xUrZk2aq74bVgAdEWdH)AY3kX2e8V3HTDCTT1S3Xo27AVAIWGzeSmNy8jYnE2JEIjpATtob54h7KtMz1p8I)3
```

**Comment by jtsorlinis** [comment]
Yeah it looks like the "Show On" field for auras isn't working properly. My auras that have "Always" are disappearing when the buff is missing

**Comment by mrbuds** [comment]
I striped the example down to be more simple

```
!WA:2!TBv3ZTXvvCBVtAs1dmjojMy30IsPrjXqdXkjUTPajsYY1o1XXSYXPPKeTFDLUR9Q9U5ExjzzkqqL26aneqLW3FvrHskfOOHmaTfMm5VGt8Wm8U5vEGj)daN7DxzNpaMbEHxy8yT3po3Z9CpNFNFNtVZ1Vi9yKA9xPFN(Dw(rwXY0EbholihZJXpUMM2mAhy)hUDiZUgHlCz(7O3xHTQfJ7q4zJLnyZz9CxAjtUtYzzmVq3GvmRgsz8tgeINqKWksEkVXjlvsqc7TnNug3z2gbe9YCw1G2rsuWDjYgUz04irFWorZISNE7TxnDHTPhrl(e5Dktc2sHlu1KtsoEvpVKNM6gs4M2QREuDrOjpmHvjxFxbnrw8tyIMHC3YLX3ZqP4XdF52oeRQLkjnj(e5NAMXp1uzdL2NzvUz6MIaIN3KoIeRkQArQr8dlGs7UyNI5Yuy2IfMnJ(SzRIk3kWZSbHR7BwHisORe1AcIPxiDTJodNGhvVWm5NAQwv9JnHeDm9TrV2mmx8i5Yp9S51VnoKW9n9MlY7F1oU(Ly8kMYxxIwcIxjL4WEEuxN0ttQZxm2fRB6fqn1UDKohNJMJ8TzviNE(8t3PKCHcHCZqtnlBMFj3Yjsx11Ptfho)mpB5W0vY1gTxDvGsKOLPVB0Tok0)43KykKNM4xoKUryWS(mFYkoONskrrPFJliOADenLIkVz4jYwX01FCyu8aWtapj8uWrWVB9ExPT8CUriNSEmtNXTcXiUF4a6vQIORezXGH9aWrtOB7zkeYrzfiYroigUmPp6d063(ilNUTlQVcSQCBYo7XQUzz2KJ1IVqUdptfXjwQTDvriRYSKfd7d20ehZ9V83Vyp90dKc2Zk2iy2Hv3VqD3aIrWMxx0tf4ygI4f1N0v5E9bBb9lZXDyfl7XQt7i)Tyeiml11H0wTGYRhS1ttmxid6Seh50OhJMfrsTWtgPFAtCOGv13HcBlLv0I9bFkkS1baDkS9ZNcgy5TUAvb5zKPofTrzdPC0qpPVxddi5Wbpm86qNBDLBDj46WFa)7DU1vscDG)i87LtcgeEn871HojH3f37I4)nXPV3TUshuRfdJsIPTLjefOS6N0)Mc1Nm4dQgbdwdDBPGrPm5KbbkSt4H5v9v4B9q3kehyxWJUcIzSPerKwGNh(WWJ1SloP)rGhA4DODP)8MEHxLJ6BA05y0sMVPsEgYk9bh5jFQdVQnRILRp5erQQT0uMIvNicHpiSd4rGpeS7esFYo0r8S9czrYhwWwKgyzIpH7AhD7gRCxtv6jJxDZgcPvPE5YLkB7qd2E3zEmHGvcrqHCMhvjKn1KxMGV44BFf5ICIeC76xw5iGZcLA3vf4ndjta7kyWCXaQKZWzL5eHi5Eli5v2xNOOCrjsbo3kCKUqTH0JSAg0Lwg1CsfyrDBIUBAapucyqD1COy)J0uURKgYOLksk9hgTwtCPZg2DkRiZcCH6ORVJJRy(Q(QqBAt)gbBXunE2iAPtWCiVDpWEV6TxGqcYiZ9c1LXpdyyypRkcDTxOXyXHukpUmHgCOHhagCfKcrwsrsY77uko7SOmBvBL4jmfHLwmTpcN7Y0hxMPynxHRLhXO7PjiNpSzyWwsLRwYIsCltdN5bgj9b3F6ddFILhGJEmz6HKuq5lKqtofYGVEClpcFMmtLzSjNU7(kUeJSOhtrK0m2pBtHSY58ioO0RPppuFEu4tIlK6JyShd(sihiwHOp4yPGlTr4OFaJnIQJ5PuhVuezGCCBhxS0HnYL1qoD11IOl4JGdJwb1cIj8WDvX7AeQRTh5uUrH)asOLzyOhHg0)AMErtFNidU96kddVRE37I22LHJo6LnErJxc50WJNXzEe9rCioNWCrkCyjrMIpwYop6gTcqKkjmBjteBkPOF6Xv7IK2RrHFNYdFp47l3XsO4CTCReW4Hn7sMAaF8HtPJjY2lmkmWgAwZK7AIr32m)UPh61m9Qs04yIMpMOnuk4nmAgGnAq4HnW60cZqjCJKcEDufAq7Ss8m8gAWpzyuwk8tXGVehsIMbVzwznadD7Un1ODVpCxFk8GWBbJKaEGeW(HpMYX21GWuaeXtmYUeJvr2q0b7ygFw8K9bhyJwcsf8g1hz)hy)hiyBDFSYAfJ5kKVpKnVKbOjBFclvZZHeEOd9MDLu2hdfECn96UoH0jo25)B9Muwh6JoXnMFQEuLKs3XTu(YZvP6y79m7fUMX62G5I9TAqmLsuLUH3zp9jZRLnJmj(2d27KYgjqqi5SY5IZQ4xkobZRrXycgIIFb2xQoyhg2K8sMCHrWoCl7Z4K4E5uRMNZzCHb8M0LtdJTZEG8W4WZOQFo)73tpX1pHjmGjHJdpBCXrykkCcyAznmSy2jHzIlOTTuqbv9TTpTIfhiqjaZSOi)eYBopfwGcbW5G6k6RQgqfQKXwsYciEsY3UlGHqWq4caheksXYgQkdiNiuZqrvV7uWIXuEqdyj4ZOy1GFN0xDFOEf2wIKF67VHL)PY9zPWNlM0d(8LGVGg(9LGMkIm4fnGViYyDrn4LVtok4vKz4WYswj4sWxso5lBaVQA1lht8aFf1Vx5oOAGVZDLfFNeoWxtj93IcVMAWx3a(MuOLA83Gcxvn4BlDbF1(GVlfdHWpa(HyqhEpd49nWgeEx432h8ZvzjWBhLvKUZ0xyc7625MiDMqesEmRhlkkF91bQpUg8RKzcqhfq)xQstG3b(1RH5BfJ5tMwMJ9lmKPzWVHkZ5UMb8wgWlGCddTwsn8Jvz0QSFzom8JufCUxe3n23X20)9iUOEPGzPkGIeu9Cu4myFkggWNoQuUefHDYOGqjhgMdonyou2roWHo06DVi7icopEmBWbHLNdoLr3Es(xJMhlZ)hl))cS8zQp)XZnxP5ypVelFd3)Ae96)ryzEjoH4WQ8VbjpCesEd3psEDi(p7olAbxRB5QyOE)1(tp3)4d
```

It seems to be an issue with Crusade (which is not a base spell) but isn't with Blessing of Freedom

**Comment by Bluehat867** [comment]
Same issue, noticed certain WA's simply do not work at all since 5.14.0.  These are not fancy or code heavy WA's, most are simple ones found from Weak Aura itself, simple heads up displays that show CDs or auras.  They just disappear when the CD is available or don't track the CD timers any more.

Paladin WA's that do not work currently but did in prior version:

- Eye of Tyr
- Divine Toll
- Divine Shield
- Holy Tracker WA:  each time you use a Holy Power ability the tracker compresses in size and when five Holy Power is reached it turn to zero.


Warrior WA's also suffering same issues::

- Rage meter
- Thunderous Roar
- Warbreaker
- Whirlwind
- Shockwave
- Spell Reflect
- Spell Block
- Shield Block
- Revenge
- Thunder Clap
- Demoralizing Shout
- Treemouth's Festering Splinter

**Comment by mrbuds** [comment]
Giving a list of your auras without an export and reproduction steps isn't helping, 

If you have a different issue open a new ticket

**Comment by submarines-and** [comment]
Also affects racial blood fury in classic era. In this case an aura that shows an icon if no cd or cd less than 5 seconds.

Keeps showing until I leave combat. Next combat it is gone.

I have the same aura for bloodrage, but that one works fine, so it seems to only affects non-standard class spells like you said. 

```
!WA:2!1vvZYTrXt8O4IYPwsvjrjLOC(hsucog7)vifX4yk(SIwJeyqrYSso2qiOD2DNv7eVANHzM12k3qN8vuvCLd6mN0JqEc6sffpa(ah5GFcONvFycG1bPE6Uv)D)RZvjFN8b5doALbmFEsdEQ0NEJZ54ZJ5YVCo8ZqsWZtvAAWJjhE(tFWsoVSB9WqfvFXxBijXpIl3IZs0EBuUwZYoJ8584a(bjnoGjOUVC6ZYbTPrsIVMXtuR7O0eP2YlKLWurw24pAREAjRDBQu9lljNqE0ve5L0oeuEs7wCbvs0Cz(pEuBAcvY8BeXpOEYXQSF2yIRowL6r3NMO3ssdzh60yRYvR2FMzY)WbbuV0WWMDr79fLRUvLTRokvrBntf3zMObQh7WHT2OuJMTA0SKttNm(IfM6TIBj5TLuLQ4Yne044vStXKXtet6sLJKusCg7AKo0b2XCEqXkPYUz(tnvGBVmYnduwIRyK8kPNRTgJuNmvCsq1vw9nkPXY5EU9NzLlS131pnzsLZAj4s)bSiCxyb46WYOm4TCHRmCCPYHsc6clzb3ckc32fUJf8)GBaVjCt4QW1G32fkaVbSI1WaM65PjyxBF6QKKUI8(4yaVtZXoPkVnZxCVWmn4jlRxXQi(rs1PYKIlRF6dEwrUSO(PR(SvSOjbwwy(rYS2el8yEa9xp3j48dvMqIFc29rd9ZNShLkkHzMV2HGMoQpjH1XqLSo86vEjLOOn0sAsBD08WI2j8e6OGuzMgTmLlPIIZ1bQEgvnDAynBt39jWIEcSDr1W7dRbpewhEV5Hpyajwer2knwrHlwXWAXjsrTW5ffrBSonAaoz4qBBMI)WcWIJWNEyBOTKNMeC01f3wtpu3k7RqUed5wIwAwhAl0P(mtYnNyH)LoQjeWA9Nj7o3vikEwwRdpOLjECpotOIghMTfAxA7M1fxnJjjf7uyfXFhwGoYUe(8KXMIDiniJ5pkw8S8qmTnXVBRqCKv6(YmfmqaSWUW8dKCn68Mit7A1RvwC5Xrreb3i2idcjxUCZblkXYJrKyLZYnbDXjAMFlDe2uIWvQUdY06uGO(t(x4s3LRizVO4xNscWCNuSztr()MJ3DmSuUxH53mL5c)hrWychtGemkt0bCzWosIO3otigN57JDoVyQ7Ke9Vb7nO(2iQxRQLR0uCZZkjh)k3Ozzsg0d(pRUzTYNYTb7f0fgn24ttMjpNMghTa815GhLFx4RgxITHnGpdkdv(rOvo4lG5HnHVe(uS2dvHVphulRgc1HTGgqtWjhSJlSn8e4PW3b7IQ)T5GsUWZwaECx4ZJ8IOS2r6pbrQib)UeXcXbBrbpfsftTvyxXttIreWcoDsJ1mlBZwAbi0sCzMM2XS7r)HuMqqdwc8dEeewi4rUhpdVBVee0mkdaZpMOuUdnKtCKBVmCyIp11jtkAc5oLCC2SUd6cBJKcG)Q1L(dYSaVJhr7o4ulFHB9CBfwhhhrOoLMC5k7qwKNk7sNhRJGl1)tfyjrdKzl3MgKT5Y4XIjW7Jpr()VX5o)qwY4gkQxHHiPpTS5MGYf66kpCYXXlnc3)WoxDr2jplp0yHS2w2VGZ7KtCTPhgnlrFgtrWXRGiNdmBLFYan3F)XqHx4E)5DybIRD6PJIVtXDisjJlhgkryFehexgMtKhZLPhLqyvhd6WjJNEQy0ZKsEn2WPC5AozODZTAklyyq76RxNRQPctW2TXRiiNb6KLf3xFjh)iQ)ERdxAUE7tKmtGABUI4SpjoLoN0pIK0MIkc)KBpSAHt26UWh1BAcg1Fg6emFEL3dV)dw7(VB(9)TD)R
```

**Comment by OldWorldRaiders** [comment]
> Giving a list of your auras without an export and reproduction steps isn't helping,
> 
> If you have a different issue open a new ticket

Hello,

Are you addressing me, or someone else?

**Comment by mrbuds** [comment]
> Are you addressing me, or someone else?

No it was addressed at the people hijacking your ticket with their (maybe unrelated) issues

**Comment by OldWorldRaiders** [comment]
> > Are you addressing me, or someone else?
> 
> No it was addressed at the people hijacking your ticket with their (maybe unrelated) issues

Understood! I was confused for a second, haha.

**Comment by Andf** [comment]
I'm having the same problem and it seems to affect all my WAs using the condition shown below, even when creating a premade aura from scratch.

![image](https://github.com/WeakAuras/WeakAuras2/assets/262401/e1a00102-9269-4132-a233-720077606bf2)

Downgrading to 5.13.2 seems to fix the issue.

**Comment by Andf** [comment]
Adding a simple way to reproduce it.
Create a premade weakaura, step 1 to 5 in screenshot 1.
Rsults shown in screenshot 2.
Weakaura is not affected by Condition number 2.

Happy to run more tests!

1-  ![WA 1](https://github.com/WeakAuras/WeakAuras2/assets/262401/8f375a58-3a0d-4dd4-a06e-a5743545d1c8)
2- ![WA 2](https://github.com/WeakAuras/WeakAuras2/assets/262401/d8b5c4e4-8b04-4a80-aee9-8a904822a15b)

**Comment by Andf** [comment]
Test above wass run with a retri pally, talent loadout:

BYEAomTTpSA9oX6huYLb5nP3rDAAAQAgIUKtSSr0koIJpIJAAAAAAQaQSCBkQkSESCRKJaRiIIiSBAA


Please note that this morning I had similar issues with a shammy enhancement in TWW beta and last night with a shadow priest on retail (different spells, of course).
